### PR TITLE
Wk 84 landing page

### PR DESCRIPTION
**Pull request by BennievanderWel**

### CODE REVIEW

File: src/styles/styles.scss [hunk 1]
```diff
@@ -1 +1,7 @@
-@import 'colors';
+@import '_colors';
+@import '_extensions';
+@import '_variables';
+
+button {
```
Comment (RRMoelker) [inline src/styles/styles.scss]: This is a weird rule. A button should not have a pointer cursor. Target specific button that should do this

File: src/views/landing/LandingPage.module.scss [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    margin-top: -6px;
+  }
+}
+
+.Logo {
+  height: 70px;
+  margin-right: 40px;
+}
+
+.HeroContainer {
```
Comment (RRMoelker) [inline src/views/landing/LandingPage.module.scss]: Code is not really Hero related

File: src/views/landing/LandingPage.module.scss [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+.Content {
+  width: $content-width;
+  box-sizing: border-box;
+  margin: -80px auto 0 auto;
+  padding: 20px 180px 0 180px;
+  background-color: white;
+  position: relative;
+
+  p:first-of-type {
```
Comment (RRMoelker) [inline src/views/landing/LandingPage.module.scss]: Seems like this should be a different class or some sort of header

File: src/styles/styles.scss (hunk 1 above)
Comment (BennievanderWel) [reply src/styles/styles.scss]: Agreed and fixed

File: src/views/landing/LandingPage.module.scss (hunk 3 above)
Comment (BennievanderWel) [reply src/views/landing/LandingPage.module.scss]: Yep, agreed and fixed.

**Pull request merged**